configs/accelerate_2gpu.yaml

```diff
@@ -2,7 +2,7 @@ compute_environment: LOCAL_MACHINE
 debug: false
 distributed_type: MULTI_GPU
 downcast_bf16: 'no'
-gpu_ids: "0,1"
+gpu_ids: "0,2"
```

Comment:
Want to leave it as 0,1

Reply:
Not so easy - it depends on what GPUs are available. But probably I should control it by CUDA_VISIBLE_DEVICES. I would change it.